reason_4.0/lib/core/config/entity_delegates/config.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+reason_include_once('classes/entity_delegates_config.php');
+
+/* $entity_delegates_config_setting = array(
+	'image' => array(
+		'append' => array(
+			'entity_delegates/image.php',
+			'entity_delegates/image_replace_original.php',
+		),
+	),
+	'site' => array(
+		'append' => array(
+			'entity_delegates/site.php',
+		),
+	),
+	'publication_type' => array(
+		'append' => array(
+			'entity_delegates/publication.php',
+		),
+	),
+); */
+
+$entity_delegates_config_setting = array(
+	'image' => array(
+		'append' => array(
+			'entity_delegates/image.php',
+		),
+	),
+	'asset' => array(
+		'append' => array(
+			'entity_delegates/asset.php',
+		),
+	),
+	'event_type' => array(
+		'append' => array(
+			'entity_delegates/event.php',
+		),
+	),
+	'publication_type' => array(
+		'append' => array(
+			'entity_delegates/publication.php',
+		),
+	),
+	'feature_type' => array(
+		'append' => array(
+			'entity_delegates/feature.php',
+		),
+	),
+	'field' => array(
+		'append' => array(
+			'entity_delegates/field.php',
+		),
+	),
+	'social_account_type' => array(
+		'append' => array(
+			'entity_delegates/social_account.php',
+		),
+	),
+	'av' => array(
+		'append' => array(
+			'entity_delegates/media_work.php',
+		),
+	),
+);
+
+
+$config = get_entity_delegates_config();
+
+$config->process($entity_delegates_config_setting);
+
```

Comment:
This is more of a question: I am not sure I understand what will happen when we use the "core" `entity_delegates_config_setting` defined above. I see that lots of the built in delegated define a functions `get_display_name`. What implementation will be used for a given entity? I see that some of those implementation check for a parent of a certain type but I don't follow which function will "win".